apps/api-gateway/src/core/observability/metrics.middleware.ts

```
@@ -0,0 +1,14 @@
+import { Request, Response, NextFunction } from 'express';
+
+export class MetricsMiddleware {
+  use(req: Request, res: Response, next: NextFunction) {
+    const start = process.hrtime.bigint();
+    res.on('finish', () => {
+      const end = process.hrtime.bigint();
+      const ms = Number(end - start) / 1_000_000;
```

ESLint

[error] 'ms' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)